packages/framework/src/Resources/translations/messages.cs.po

```diff
@@ -31,6 +31,9 @@ msgstr "-- Vyberte značku --"
 msgid "-- Choose unit --"
 msgstr "-- Vyberte jednotku --"
 
+msgid "-- Select name of the transfer --"
+msgstr "-- Vyber název převodu --"
```

Comment:
usually, we use the polite form (`vyberte`)

Reply:
c57add06f8